packages/shared/src/components/streak/ReadingStreakButton.tsx

```diff
@@ -86,31 +89,52 @@ export function ReadingStreakButton({
   }
 
   return (
-    <Tooltip
-      content="Current streak"
-      streak={streak}
-      shouldShowStreaks={shouldShowStreaks}
-      setShouldShowStreaks={setShouldShowStreaks}
-    >
-      <Button
-        type="button"
-        icon={<ReadingStreakIcon secondary={hasReadToday} />}
-        variant={
-          isNewMobileLayout && isMobile
-            ? ButtonVariant.Tertiary
-            : ButtonVariant.Float
-        }
-        onClick={handleToggle}
-        className={classnames('gap-1', compact && 'text-theme-color-bacon')}
-        size={
-          (isLaptop || !compact) && !isMobile
-            ? ButtonSize.Medium
-            : ButtonSize.Small
-        }
+    <>
+      <ConditionalWrapper
+        condition={!isMobile}
```

Comment:
Since there is content when it is not compact, I think we should also include the condition.
```suggestion
        condition={!isMobile && compact}
```

Reply:
Not sure I follow your comment here, the conditional wrapper is only for the `Tooltip`, which dictates wether it's wrapped around the button on non-mobile resolutions. It should have nothing to do with the `compact` property 🤔

Reply:
Yep, but the thing is, we only display tooltip for buttons with text. Which when compact is applied, there is no text, and only when should be applied.

Reply:
but tooltips are not a thing on mobile, right? so including `compact` here would not really matter. 

Also, since the `Tooltip` or `CustomStreaksTooltip` is not dictated by `compact`, I feel like we could mistakenly create a situation where someone uses the `compact` version and ends up displaying both the main streaks tooltip and the drawer. I would suggest we tinker with this only if the situation arises that requires it, WDYT?

Reply:
Ah, okay, I misunderstood, I thought this is only for tooltip to display a text. In that case, feel free to ignore and move on.